src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -607,6 +612,10 @@ int fx_muxer_t::parse_args_and_execute(
         known_opts.push_back(_X("--depsfile"));
         known_opts.push_back(_X("--runtimeconfig"));
     }
+    if (mode == host_mode_t::muxer)
```

Comment:
Help is only for the muxer. The help function will be respected only for the muxer itself. (the function is called `muxer_usage`)